Library/Homebrew/cmd/update.rb

```diff
@@ -22,9 +22,11 @@ def update
     migrate_taps
 
     report = Report.new
+    repo_changes = []
     master_updater = Updater.new(HOMEBREW_REPOSITORY)
     master_updater.pull!
     report.update(master_updater.report)
+    repo_changes << ["homebrew/homebrew", *master_updater.revision_range]
```

Comment:
I don’t think it’s necessary to expand the second part, i.e. you can have `[name, [a, b]]` instead of `[name, a, b]`.

`a, b, c = [1, [2, 3]]` doesn’t work as expected in Ruby, but it’s better not to use multiple assignments anyway because they create an array behind the scene.